elasticsearch/main.tf

```diff
@@ -91,8 +91,11 @@ resource "aws_elasticsearch_domain" "es" {
   }
 
   domain_endpoint_options {
-    enforce_https       = var.endpoint_enforce_https
-    tls_security_policy = var.endpoint_tls_security_policy
+    enforce_https                   = var.endpoint_enforce_https
+    tls_security_policy             = var.endpoint_tls_security_policy
+    custom_endpoint_enabled         = var.custom_endpoint == null ? false : true
+    custom_endpoint                 = var.custom_endpoint == null ? null : var.custom_endpoint
```

Comment:
Wouldn't it be cleaner to just use `custom_endpoint = var.custom_endpoint`? 😅 
And the same for the cert arn below.

Reply:
oops! definitely yes